### PR TITLE
add segmented review FAQ

### PR DESCRIPTION
**Pull request by aviv**
cf. https://github.com/dobtco/screendoor-v2/pull/1882

### CODE REVIEW

File: contents/articles/screendoor/evaluation/configuring_evaluation_fields.md [hunk 1]
```diff
@@ -45,3 +45,8 @@ To associate a rating field with a specific response field, select the appropria
 After you connect a rating field to a response field, the rating will show up next to the response field (rather than under "Rate this response") when you evaluate responses.
 
 ![rating field connected to response field](../images/rating_field_connected.png)
+
+## Can I make a rating field that only a specific team can see?
+Yes! To only show a rating field to a specific team, click the "Only show to..." link for that rating field and select the appropriate team from the dropdown. Any collaborators who aren't on that team won't be able to see that rating field or evaluate based on it.
```
Comment (ajb) [inline contents/articles/screendoor/evaluation/configuring_evaluation_fields.md]: I think we should mention that you need to configure teams first. I don't really think this belongs as a FAQ, either.

**Comment by aviv** [comment]
OK, changed it up. We could also move this to the "Project teams" article, but I don't think it'd really belong there.

**Comment by ajb** [comment]
:+1:

cc @kariannemah for your demo.

**Comment by kariannemah** [comment]
ty both!

**Pull request merged**